package/gomod/mod.go

```diff
@@ -102,6 +102,9 @@ func parse(opt *option, path string, data []byte) (*Module, error) {
 	if err != nil {
 		return nil, err
 	}
+	if modfile.Module == nil {
+		return nil, fmt.Errorf("mod: missing module statement in %q", path)
+	}
```

Comment:
Fixes periodic panics in CI. Still trying to get to the bottom of why this sometimes occurs only in CI.

Reply:
This happens when the go.mod exists but doesn't have a `module <name>` statement